.github/workflows/release.yml

```diff
@@ -0,0 +1,24 @@
+name: goreleaser
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  goreleaser:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Set up Go
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.13.x
+      - name: Run GoReleaser
+        uses: goreleaser/goreleaser-action@v1
+        with:
+          version: latest
+          args: release --rm-dist
+        env:
+          GITHUB_TOKEN: ${{ secrets.GO_RELEASER }}
```

Comment:
@jckuester you'll need to add a GO_RELEASER secret to the repo settings. You can create a token here. https://github.com/settings/tokens

We can rename the secret if you want but that's just what I've used in the past.

Reply:
The token name is fine.